cypress/support/commands.js

```diff
@@ -29,13 +25,69 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[placeholder="${selector}"]`);
```

Comment:
The `getByDataCy` command is using the `placeholder` attribute to select elements. Ensure that this is intentional and that the elements you are targeting have the `placeholder` attribute set to the desired value. If you intended to use a `data-cy` attribute, adjust the selector accordingly.